backstop_data/engine_scripts/puppet/administer-menu/connections.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Page = require('../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, vp) => {
+  const page = new Page(engine, scenario, vp);
+
+  await page.closeErrorNotifications();
```

Comment:
every single page seems to use `page.closeErrorNotifications();`, why don't we move this to `onBefore.js` or `onReady.js` ?

Reply:
unfortunately if you provide a global "onBefore" / "onReady" script (you can do that by defining it at the root level of the config file), it will work only for those scenario without their own "onBefore" / "onReady" script. So we have to call `closeErrorNotifications` "manually" on every scenario's script (that needs the notifications to be closed, that is)